### ISSUE TITLE
How to remove Profile picture from Author slot

### ISSUE THREAD

**Issue by sanjeevba** [microsoft-search/pnp-modern-search#4098]
In PnP Search Results Webpart, I'm using the Cards layout with LocalSharePointResults.  I'm trying to remove the Capital 'W'  shown in the picture (bottom left). I've tried modifying the _Managed Card Fields_ but was not successful.

I've even tried switching to a custom handle bar template and modifying it. I realize if I don't enter a profile name it takes the first letter of whatever is in that slot. 

 Is it it possible to remove the circled Capitial W?

![image](https://github.com/user-attachments/assets/02cfb49e-5595-4d1f-97cf-67b88f78f09b)

**Comment by kasperbolarsen** [comment]
You are aware that the circle will show a picture if the UPA account has one?
![image](https://github.com/user-attachments/assets/a7d074a7-b7d6-4141-907e-625055261361)

**Comment by sanjeevba** [comment]
Yes, I am aware the picture will show if the UPA has one. In this case I'm not using that slot as a User. I'm using it as a PhotoCategory...as in Season (Fall, Winter, Summer, Spring). So the the Author slot is actually  PhotoCategory. And the Circle with the letter is the first letter of the Season. Is there a way to remove the Circle with the Letter?
